packages/dd-trace/src/plugins/util/git.js

```diff
@@ -77,6 +77,18 @@ function isDirectory (path) {
   }
 }
 
+function isGitAvailable () {
+  const isWindows = os.platform() === 'win32'
+  const command = isWindows ? 'where' : 'which'
+  try {
+    cp.execFileSync(command, ['git'], { stdio: 'pipe' })
+    return true
+  } catch (e) {
+    incrementCountMetric(TELEMETRY_GIT_COMMAND_ERRORS, { command: 'check_git', exitCode: 'missing' })
```

Comment:
⚠️ `check_git` is not standard but TBH it seems weird to use any of the other tags. In any case, when checking for git errors, we group by `command`, so this value should also appear in our dashboards.

Let me know if I'm missing something